app/graphql/mutations/update_well_position.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Mutations
+  # Mutation to handle the update of a well position for a well with a given ID.
+  class UpdateWellPosition < BaseMutation
```

Comment:
Would this be better named something like `UpdateWellPositionMutation` to more easily identify it as a mutation?

Reply:
I did do that at first.  I felt that being in the `Mutations` namespace made it unnecessary to name the class with `Mutations` as well.